### PR TITLE
Add Instructure icon

### LINKED ISSUE #7889: Instructure Icon
### Brand Name

Instructure

### Website

https://www.instructure.com/

### Popularity Metric

The Similarweb Global rank is 108. See https://www.similarweb.com/website/instructure.com/#overview

### Official Resources for Icon and Color

svg: using svg-grabber from the official website
color: `#2A7BA0`, from the official website (this is the primary color)

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by Anmol-Baranwal**
instructure_updated

**Issue:** closes #7889 

**Similarweb rank:**
Global rank: 108 &nbsp; `https://www.similarweb.com/website/instructure.com/#overview`

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG logo is retrieved from: https://www.instructure.com/
The hex value is extracted from the official website

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5348,6 +5348,11 @@
             "hex": "FABF15",
             "source": "https://www.instructables.com/community/Official-Instructables-Logos-1/"
         },
+        {
+            "title": "Instructure",
+            "hex": "2A7BA0",
+            "source": "https://www.instructure.com/"
```
Comment (LitoMore) [inline _data/simple-icons.json:5354]:
```suggestion
            "source": "https://www.instructure.com/about/brand-guide/download-logos",
            "guidelines": "https://www.instructure.com/canvas/resources/noram-guides/instructure-brand-guide-2022"
```

Their logo resource and brand usage guideline can be found here.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (Anmol-Baranwal) [reply _data/simple-icons.json]: I have made the necessary changes

**Review by LitoMore — CHANGES_REQUESTED** [review]
@Anmol-Baranwal I found a black version of their brand in their logo kit: https://www.instructure.com/about/brand-guide/download-logos.

CleanShot 2022-10-01 at 23 31 16@2x

Could you process with this one? The file can be found in:

```
Instructure/Horizontal/Black/Instructure_Logo_Horizontal_Black_RGB.svg
```

**Comment by Anmol-Baranwal** [comment]
Overlap preview:
preview_overlap

**Review by LitoMore — APPROVED** [review]
CleanShot 2022-10-03 at 19 05 56@2x

@Anmol-Baranwal Cool! Thank you!

**Pull request merged**